website/www/site/layouts/index.html

```diff
@@ -209,16 +209,24 @@ <h2 class="calendar-title">
       {{ T "home-quotes-title" }}
     </div>
 
-    <div class="quotes-desktop swiper">
-      <div class="swiper-wrapper">
-        {{ $data := index $.Site.Data .Site.Language.Lang }}
-        {{ range $quote := $data.quotes }}
-        <div class="swiper-slide">
-          {{ partial "quotes/quote.html" (dict "icon" $quote.icon "text" $quote.text  "logoUrl" $quote.logoUrl "linkUrl" $quote.linkUrl "linkText" $quote.linkText) }}
+    <div class="swiper-container">
+      <div class="swiper-button-prev-custom">
+        <img src="/images/swiper-button-prev.png" alt="previous button">
+      </div>
+      <div class="quotes-desktop swiper">
+        <div class="swiper-wrapper">
+          {{ $data := index $.Site.Data .Site.Language.Lang }}
+          {{ range $quote := $data.quotes }}
+          <div class="swiper-slide">
+            {{ partial "quotes/quote.html" (dict "icon" $quote.icon "text" $quote.text  "logoUrl" $quote.logoUrl "linkUrl" $quote.linkUrl "linkText" $quote.linkText) }}
```

Comment:
consider formatting the file

Reply:
tried to put attributes on different lines, but since it's hugo code doesn't work correctly